app/commands/codemod/confirm.go

```diff
@@ -0,0 +1,157 @@
+package codemod
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/codelingo/lingo/app/util"
+	flow "github.com/codelingo/platform/flow/rpc/flowengine"
+	"github.com/fatih/color"
+	"github.com/waigani/diffparser"
+)
+
+type IssueConfirmer struct {
+	keepAll bool
+	output  bool
+}
+
+func NewConfirmer(output, keepAll bool, d *diffparser.Diff) (*IssueConfirmer, error) {
+	cfm := IssueConfirmer{
+		keepAll: keepAll,
+		output:  output,
+	}
+
+	return &cfm, nil
+}
+
+func GetDiffRootPath(filename string) string {
+	// Get filename relative to git root folder
+	// TODO: Handle error in case of git not being installed
```

Comment:
Make sure you add this TODO as an issue, milestone, or in your project management tool; eg. Trello.